recipes/ixmp/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "0.1.0" %}
+
+package:
+  name: ixmp
+  version: {{ version }}
+
+source:
+  url: https://github.com/iiasa/ixmp/archive/v{{ version }}.tar.gz
+  sha256: 512c63bc8473e667ba8571fb7125540ce47f5245b3cca71c989f089600cfd494
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  script: python -m pip install --no-deps --ignore-installed .
```

Comment:
It looks like there is a python entry point. Please include it:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#python-entry-points

and add tests for the CLI:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#test-commands